src/main/java/com/kilopo/kosshop/DAO/Impl/AddressDAOImpl.java

```diff
@@ -0,0 +1,79 @@
+package com.kilopo.kosshop.DAO.Impl;
+
+import com.kilopo.kosshop.DAO.BaseDAO;
+import com.kilopo.kosshop.entity.Address;
+import com.kilopo.kosshop.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.query.Query;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class AddressDAOImpl implements BaseDAO<Address> {
+    public Address getById(Long id) throws SQLException {
```

Comment:
Create BaseDAOImpl and implemet CRUD operatin there, and just extend it. And do not duplicate code in all DAO's